week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -11,9 +20,13 @@ function mood() {
   }
 }
```

Comment:
What you did here is correct but you have removed the else statement inside the mood function. It should be there if the isHappy is false the else should be "I'm not happy" and true for the if "i'm happy".Fix that Sulaiman.

Reply:
I did not remove the else statement  ,,, it is inside ... I will only add ===  true inside if statement